src/docs/sdk/sessions.mdx

```diff
@@ -308,6 +308,16 @@ with one difference: sessions transitioned to `exited` have their session
 duration averaged. This lets Sentry show you the duration of non crashed
 sessions over time.
 
+## Alerts
+<Alert title="WIP" level="info">
+  Session based alerting is currently in early access.
```

Comment:
Does "early access" mean anything in develop docs?

Reply:
removed